### ISSUE TITLE
Female Focused Symptoms/ Conditions Updating 4

### ISSUE THREAD

**Issue by Lucy-WhiteSwan** [obophenotype/human-phenotype-ontology#7514]
**Preferred term label:**
Postcoital Bleeding

**Synonyms**
Bleeding after sex, PCB

**Definition (free text, please give PubMed ID)**
PUBMed ID: 25045355, Postcoital bleeding (PCB) consists of spotting or bleeding after sexual intercourse that is not related to a person's menstrual cycle.

**Parent term (use hpo.jax.org/app)**
Abnormal bleeding HP:0001892

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
n/a

**Your nano-attribution (ORCID)**
n/a

**Comment by pnrobinson** [comment]
@Lucy-WhiteSwan thanks! Adding as Postcoital vaginal bleeding, child of abnormal vaginal bleeding (newly created term).
